docs/components.rst

```diff
@@ -72,60 +78,87 @@ hardlink files will be automatically available to the user on a `JupyterLab` ins
 a volume. Any file that is found under a directory ``/wpsoutputs/<bird-name>/users/<user-id>/`` is considered to be
 user data and any outside file is considered public.
 
-The permissions found on the files are synchronized with the permissions found on `Magpie`. If `Magpie` uses a
-`secure-data-proxy` service, this service handles the permissions of those files. If a file does not have a
-corresponding route on the `secure-data-proxy` service, it will use the closest parent permissions. Note that the route
-resources found under the `secure-data-proxy` service must match exactly a path on the filesystem, starting with the
-directory name ``wpsoutputs``, and following with the desired children directories/file names. If no `secure-data-proxy`
-service is found, the user files are assumed to be fully available with read and write permissions for the user. Note
-that if the file does not have any read or write permissions, the hardlink will not be available in the user's
-workspace.
+The permissions found on the user data are synchronized with the permissions found on `Magpie`_. If `Magpie`_ uses a
+`secure-data-proxy` service, this service handles the permissions of those files (see `here <https://github.com/
```

Comment:
All ``` `secure-data-proxy` ``` should be with double ``` `` ```, otherwise they appear as plain text (see https://github.com/Ouranosinc/cowbird/blob/5595572ce915c6f66ca4dcf51dd4bb07986e0786/docs/components.rst).